### PR TITLE
[TDV-133]  Token trailing digits count

### PR DESCRIPTION
**Pull request by EddyRock**
https://tokend.atlassian.net/browse/TDV-133

### CODE REVIEW

**Comment by letehaha** [comment]
[6f6f104](https://github.com/tokend/web-client/pull/269/commits/6f6f10479ecf495326de5557de1fcf7ee6f6196f) and [37616a2](https://github.com/tokend/web-client/pull/269/commits/37616a2883ec818ea215d2845e85e6154c01aad5) – commit message does not equal to commit diff. Pay attention and not allow it for after

File: src/vue/forms/LimitsForm.vue [hunk 1]
```diff
@@ -234,11 +253,7 @@ export default {
   name: 'limits-form',
   mixins: [FormMixin],
   props: {
-    limits: {
-      type: Object,
-      required: true,
-      default: () => ({}),
-    },
+    limits: { type: Object, default: () => ({}) },
```
Comment (letehaha) [inline src/vue/forms/LimitsForm.vue]: Why it does not required now?

File: src/vue/forms/TransferForm.vue [hunk 2]
```diff
@@ -379,6 +385,12 @@ export default {
     try {
       await this.loadCurrentBalances()
       this.setAsset()
+      const asset = this.assets.find(asset => {
```
Comment (letehaha) [inline src/vue/forms/TransferForm.vue]: `this.assets.find(asset => asset.code === this.assetToTransfer)`
Pay attention

File: src/vue/forms/WithdrawalForm.vue [hunk 3]
```diff
@@ -277,9 +279,13 @@ export default {
...
 
-    selectedAssetStep () {
-      // eslint-disable-next-line
-      return inputStepByDigitsCount(this.form.asset.trailingDigitsCount) || config.MIN_AMOUNT
+    trailingDigitsCount () {
+      return this.assets
+        .find(asset => asset.code === this.form.asset.code)
+        .trailingDigitsCount || config.DECIMAL_POINTS
+    },
+    minValue () {
```
Comment (letehaha) [inline src/vue/forms/WithdrawalForm.vue]: This computed do nothing. Remove it and in the 57 line use just `config.MINIMAL_NUMBER_INPUT_STEP`

File: src/vue/forms/market-orders/CreateTradeOfferForm.vue [hunk 4]
```diff
@@ -173,18 +180,29 @@ export default {
...
+    },
+    accountAssets () {
+      return this.accountBalances
+        .map(balance => balance.asset)
+        .filter(asset => asset !== this.assetPair.quote)
+    },
+    trailingDigitsCount () {
+      return this.assets
+        .find(asset => asset.code === this.assetPair.base)
+        .trailingDigitsCount || config.MIN_AMOUNT
```
Comment (letehaha) [inline src/vue/forms/market-orders/CreateTradeOfferForm.vue]: This line looks really strange, make it pretty

Comment (letehaha) [reply src/vue/forms/market-orders/CreateTradeOfferForm.vue]: Also, you have a `selectedAssetStep` that make this `||` instead of you

**Comment by max-nes** [comment]
Outdated

**Pull request closed without merging**